app/src/main/java/net/squanchy/proximity/near/NearITModule.java

```diff
@@ -0,0 +1,17 @@
+package net.squanchy.proximity.near;
+
+import net.squanchy.proximity.ProximityProvider;
+import net.squanchy.proximity.ProximityService;
+
+import dagger.Module;
+import dagger.Provides;
+
+@Module
+public class NearITModule {
```

Comment:
This is leaking implementation details, should probably be `ProximityModule` (since it offers a `ProximityService`, which happens to be implemented by `NearITProximityService`)